.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
-  - "10"
+  - '10'
+
+
+  script: npm run verify
```

Comment:
It seems there are no longer any differences in this file, so lets just revert the changes here.

Reply:
Done.